src/pages/image/[uuid].astro

```diff
@@ -70,43 +119,110 @@ const tissueY = (1e6 * image.PhysicalSizeY * image.sizeY).toPrecision(2);
                 </a>
             </div>
         </section>
-        { ( image.source_image_uuid != null ) &&
+
+        { ( image.canconical_representation != null ) &&
```

Comment:
canonical_representation (typo?): couldn't find any info on this in the image json.

Reply:
I think i'm just going to remove this for now - we don't yet deal with derived images